content/docs/user-guide/how-to/best-practices.md

```diff
@@ -0,0 +1,136 @@
+# Best Practices for DVC Projects
+
+Asking questions on data science collaboration to data scientists, engineers, or
+managers, we'll get a variety of answers. DVC provides a systematic approach
+towards managing and collaborating on data science projects. You can manage your
+projects with DVC more efficiently using the practices listed here:
+
+- Source code and data versioning
+
+  You can use DVC to avoid discrepancies between
+  [revisions](https://git-scm.com/docs/revisions) of source code and versions of
+  data files, when the data doesn't fit into a traditional repository. DVC
```

Comment:
Nice. You changed the open question into an answer. Let's just be much more on point for how-tos. Some distracting notes should be omitted and you can link to other docs so please get very familiar with all of the existing docs! For example:

```suggestion
- Source code and data versioning

  You can use DVC to avoid discrepancies between
  [revisions](https://git-scm.com/docs/revisions) of source code and
  [versions](/doc/use-cases/versioning-data-and-model-files) of data files. DVC
```
> Needs formatting.

Reply:
BUT this point is still mostly explanation and not very precise solution. See https://documentation.divio.com/introduction/ again to have a clear understanding of the differences.

If you think Best Practices should be a regular guide and not a How-to, we can discuss that. Please don't just follow my instructions blindly, I'm expecting to be wrong sometimes and for you to drive this whole process!

Reply:
Yea as I described in https://github.com/iterative/dvc.org/pull/1705#discussion_r479693987 its better to make it a regular guide. It can have some explanationa and example. if an example already exist that can be linked too.

Reply:
OK. I'm not sure but I'll let you lead this process. Curious to see what links you would add. As mentioned before, please make a new draft PR for that and leave your suggested changes there. Maybe I'll take it over then, as it's slightly advanced and not necessarily related to your GSoD project description (1.x updates and how-to).